code/index.html

```diff
@@ -1,22 +1,90 @@
 <!DOCTYPE html>
+
 <html>
+
   <head>
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
   </head>
+  
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+<div class="webpage">
```

Comment:
This tag is redundant, if you want some styling that applies to your whole page - you can use the `<body>` for that! Also, remember indentation (indent for every nested element). Take a look at the [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) if you want to recap :)